api/swagger-spec/oapi-v1.json

```diff
@@ -17354,6 +17354,10 @@
       },
       "description": "Triggers determine how new Builds can be launched from a BuildConfig. If no triggers are defined, a new build can only occur as a result of an explicit client build creation."
      },
+     "schedulePolicy": {
+      "$ref": "v1.BuildSchedulePolicy",
+      "description": "SchedulePolicy describes how the new build created from this build configuration will be scheduled for execution. This is optional, if not specified we default to \"parallel\"."
```

Comment:
@smarterclayton should we still default to Serial or SerialLatestOnly? I think Serial will be "safe" default.